WireDomain/Sources/WireDomain/Event Processing/ConversationEventProcessor/ConversationCreateEventProcessor.swift

```diff
@@ -26,15 +27,33 @@ protocol ConversationCreateEventProcessorProtocol {
     ///
     /// - Parameter event: A conversation create event.
 
-    func processEvent(_ event: ConversationCreateEvent) async throws
+    func processEvent(_ event: ConversationCreateEvent) async
 
 }
 
 struct ConversationCreateEventProcessor: ConversationCreateEventProcessorProtocol {
 
-    func processEvent(_: ConversationCreateEvent) async throws {
-        // TODO: [WPB-10166]
-        assertionFailure("not implemented yet")
+    let repository: any ConversationRepositoryProtocol
+
+    func processEvent(_ event: ConversationCreateEvent) async {
```

Comment:
Usually when a conversation is created we add a system message "Alice started a conversation...". I would expect to see that here after the conversation was stored locally.

Reply:
it's done directly in the ConversationLocalStore:

```
/// we just got a new conversation, we display new conversation header
conversation.appendNewConversationSystemMessage(
    at: .distantPast,
    users: conversation.localParticipants
)
```

in another PR, I factored out this helper method and expose it through the repository and local store so we can create any ZMSystemMessage and append it to a given conversation (so pasted code above will also be refactored when the related PR is merged), you can check it out here (line 279 I added a comment)

https://github.com/wireapp/wire-ios/pull/2041/files

Reply:
ok great 👍🏻